src/schema/schema_validators.py

```diff
@@ -503,6 +562,13 @@ def validate_creation_action(property_key, normalized_entity_type, request, exis
     if creation_action == '':
         raise ValueError(f"The property {property_key} cannot be empty, when specified.")
 
+    if creation_action == 'external process':
```

Comment:
On line 559 we need to set `creation_action` to lower other this code will not operate correctly. Will need to remove `.lower()` on line 560